src/test/java/io/zold/api/WalletTest.java

```diff
@@ -85,8 +93,11 @@ private Path wallet(final long id) throws IOException {
                     "zold",
                     "1",
                     hex,
-                    // @checkstyle LineLength (1 line)
-                    "MIGeMA0GCSqGSIb3DQEBAQUAA4GMADCBiAKBgGZCr/9hBChqsChd4sRAIpKNRinjhSW+J+S7PU5malVMiRHVoKjeooLDpWpij0A6vkzOvjrMldAZT0Fzgp0cJ15TOVwiQanQ5WuQDgRkLoxrdh/qyBApoDvk4OUEozOQPNwfpZOFfaUALPsPnv9995TlY9WcdSKW5dj041p1tJmlAgMBAAE="
+                    // @checkstyle LineLength (5 lines)
+                    "MIGeMA0GCSqGSIb3DQEBAQUAA4GMADCBiAKBgGZCr/9hBChqsChd4sRAIpKNRinjhSW+J+S7PU5malVMiRHVoKjeooLDpWpij0A6vkzOvjrMldAZT0Fzgp0cJ15TOVwiQanQ5WuQDgRkLoxrdh/qyBApoDvk4OUEozOQPNwfpZOFfaUALPsPnv9995TlY9WcdSKW5dj041p1tJmlAgMBAAE=",
```

Comment:
@t-izbassar this file is eventually going to become very cumbersome. Let's move it to a static file in `src/test/resources`. That way we can add additional wallet files for different tests.